src/scss/components/_input.scss

```diff
@@ -1,12 +1,15 @@
+$dp-input-border-size: 1px;
+$dp-input-padding-size: 0.5rem;
+$numeric-character-size: 0.6rem;
+
 .dp-input {
   display: block;
   position: relative;
-  padding: 0.5rem;
+  padding: $dp-input-padding-size;
   width: 100%;
   z-index: 3;
```

Comment:
`z-index` here feels a bit arbitrary—is there a reason for it?

Reply:
Not sure. It was in the DST design system's input class, so I ported it over. Assuming there was reasoning behind its inclusion originally. That said, happy to remove it if we think it's unnecessary here

Reply:
Have now removed the z-index rule